scripts/quick_start.sh

```diff
@@ -48,7 +46,15 @@ run_container() {
     echo container started
 }
 
-stop_all
-docker network create --subnet=172.20.0.0/24 xline_net >/dev/null 2>&1
-
-run_container 3
+if [ -z "$1" ]; then
```

Comment:
use `"$1" == "start"` ?

Reply:
No parameter means start, does not destroy the original way of use